OpenOversight/app/models/database.py

```diff
@@ -468,8 +468,7 @@ class Incident(BaseModel):
     __tablename__ = "incidents"
 
     id = db.Column(db.Integer, primary_key=True)
-    date = db.Column(db.Date, unique=False, index=True)
-    time = db.Column(db.Time, unique=False, index=True)
+    occurred_at = db.Column(db.DateTime(timezone=True), unique=False, nullable=False)
```

Comment:
This should be a nullable field. Not every incident has a date or time associated with it.
Actually there is a bigger problem, how can we decide between an incident for which no time is known and an incident that happened at midnight. I think that is actually the reason these were two different fields. Sorry, I wasn't really thinking about this issue before. Maybe an additional field of `time_unknown` could help? Or more interestingly, an enum that allows us to set which part of the time we know, e.g. if only the year and month is known, we only display that.
So I think I am fine with merging this, but I will think about following it up with that enum idea

Reply:
I'd be happy to make that update, if you'd like. It should require too much work and it's not like it's a feature anyone is waiting on.

Reply:
I'll modify it so that we either have a `date` or `occurred_at` column.